deploy/05_group_health.ts

```diff
@@ -0,0 +1,30 @@
+import { DeployFunction } from "@celo/staked-celo-hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { catchNotOwnerForProxy } from "../lib/deploy-utils";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deploy } = hre.deployments;
+  const { deployer } = await hre.getNamedAccounts();
+
+  await catchNotOwnerForProxy(
+    deploy("GroupHealth", {
+      from: deployer,
+      log: true,
+      proxy: {
+        proxyArgs: ["{implementation}", "{data}"],
+        proxyContract: "ERC1967Proxy",
+        execute: {
+          init: {
+            methodName: "initialize",
+            args: [hre.ethers.constants.AddressZero, deployer],
```

Comment:
Can you please explain what exactly do you mean by this comment ? This is calling initialize method which we use across all Proxied smart contracts

Reply:
I'm referring to the owner address that we set here. Instead of `deployer`, it should be set to `multiSig`. This became true since there are no dependencies to set post deployment. Reference `rebased stCELO` deployment script.

Reply:
Updated